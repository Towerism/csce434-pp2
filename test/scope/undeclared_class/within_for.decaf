int asdf() {
    int i;
    for (i = 0; i < 6; i++) {
        Asdf fdsa;
    }
}