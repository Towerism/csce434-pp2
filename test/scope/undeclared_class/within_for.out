
*** Error line 4.
        Asdf fdsa;
        ^^^^
*** No declaration found for type 'Asdf'

